src/components/grid-aware/Covid19InfoBoxBlock/Covid19InfoBoxBlock.module.css

```diff
@@ -0,0 +1,33 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  background: var(--color-gray-300);
+  grid-column: 2;
+  padding: 60px;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    display: block;
+  }
+
+  .bleedMainContent {
+    padding: 50px 40px;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  font-size: 30px;
+}
+
+.description {
+  font: var(--font-body-medium);
+  font-size: 20px;
```

Comment:
Same with this, except it doesn't look like we have an appropriate font typography in mobile, so instead, could you only set this `font-size` under a media query?

```suggestion
```

Reply:
👍 Deleted and moved to media query.